src/userDashboard/userDashboard_Top/UserDashboard_Top.js

```diff
@@ -0,0 +1,61 @@
+import { OPTIONS, USER_DASHBOARD_TITLES } from "../ConstantsUserDashboard";
+import * as s from "../userDashboard_Styled/UserDashboard.sc";
+import IntervalDropdownList from "./IntervalDropdownList";
+import TimeFormatDropdownList from "./TimeFormatDropdownList";
+import UserDashboardTabs from "./UserDashboardTabs";
+import UserDashboardDatePicker from "./UserDashboardDatePicker";
+
+export default function UserDashboard_Top({
+  activeTab,
+  handleActiveTabChange,
+  activeTimeInterval,
+  handleActiveTimeIntervalChange,
+  handleActiveTimeFormatChange,
+  activeTimeFormatOption,
+  referenceDate,
+  setReferenceDate,
+}) {
+  return (
+    <s.UserDashboardTopContainer>
+      <s.UserDashboardTitle>
+        {USER_DASHBOARD_TITLES.MAIN_TITLE}
+      </s.UserDashboardTitle>
+
+      <UserDashboardTabs
+        activeTab={activeTab}
+        handleActiveTabChange={handleActiveTabChange}
+      />
+
+      <s.UserDashboardHelperText>
+        <>
+          {activeTab === 1
+            ? USER_DASHBOARD_TITLES.BAR_GRAPH_TEXT
+            : USER_DASHBOARD_TITLES.LINE_GRAPH_TEXT}
+
+          <IntervalDropdownList
+            handleActiveTimeIntervalChange={handleActiveTimeIntervalChange}
+            activeTimeInterval={activeTimeInterval}
+          />
+
+          {(activeTimeInterval === OPTIONS.CUSTOM_WEEK ||
+            activeTimeInterval === OPTIONS.CUSTOM_MONTH ||
+            activeTimeInterval === OPTIONS.CUSTOM_YEAR) && (
+            <UserDashboardDatePicker
+              referenceDate={referenceDate}
+              setReferenceDate={setReferenceDate}
+            />
+          )}
+          {activeTab === 1 && (

```

Comment:
can we give a symbolic name to this 1? 
magic constants make the code harder to understand!

Reply:
done